rdkit/Chem/SaltRemover.py

```diff
@@ -33,20 +33,61 @@
 
 import os
 import re
+from rdkit.Chem.rdmolfiles import SDMolSupplier
 
 from rdkit import Chem
 from rdkit import RDConfig
 from contextlib import closing
 
+class Format:
```

Comment:
It is not only about file format, rather about _data_ or _input_ format. Because, in my opinion, _data_ is too vague, lets go for `InputFormat`.